File: check-two-and-three/checkTwoAndThree.js
const checkThreeAndTwo = (array) => {
    const check = {};
    array.forEach((element) => {
        if(check[element] === undefined) check[element] = 1;
        else { check[element] += 1 };
    });

    if(Object.keys(check).length !== 2) {
        return false;
    };
    if(Object.values(check)[0] === 3 || Object.values(check)[1] === 2){
        return true
    };
    if(Object.values(check)[0] === 2 || Object.values(check)[1] === 3){
        return true
    };
    return false;

};

module.exports = { checkThreeAndTwo };

//https://www.codewars.com/kata/5a9e86705ee396d6be000091/train/javascript

File: python-venv/check_two_and_three/kata_function/check_two_and_three.py
def check_two_and_three(array):
    check = {}
    for i in range(0, len(array), 1):
        if array[i] in check:
            check[array[i]] += 1
        else:
            check[array[i]] = 1

    if len(list(check.keys())) != 2:
        return False
    elif list(check.values())[0] == 3 and list(check.values())[1] == 2:
        return True
    elif list(check.values())[0] == 2 and list(check.values())[1] == 3:
        return True
    else:
        return False


#https://www.codewars.com/kata/5a9e86705ee396d6be000091/train/python